rllib/utils/test_utils.py

```diff
@@ -1358,6 +1359,11 @@ def run_rllib_example_script_experiment(
         tune_callbacks: A list of Tune callbacks to configure with the tune.Tuner.
             In case `args.wandb_key` is provided, will append a WandB logger to this
             list.
+        keep_config: Set this to True, if you don't want this utility to change the
```

Comment:
Yeah. Very nice. Had a couple of cases where my config was changed unintendedly by the function and it was ambiguous what happened.